.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           if [[ "${{ github.event_name }}" == "pull_request" && "${{ github.base_ref }}" == "dev" && "${{ matrix.NXF_VER }}" != "latest-everything" ]]; then
             echo matrix='["latest-everything"]' | tee -a $GITHUB_OUTPUT
           else
-            echo matrix='["latest-everything", "22.10.1"]' | tee -a $GITHUB_OUTPUT
+            echo matrix='["latest-everything", "23.04.0"]' | tee -a $GITHUB_OUTPUT
```

Comment:
not caught by tools

Reply:
cc @mirpedrol

Reply:
@sateeshperi I'm assuming this is your magic

Reply:
The version is updated in the current dev version of tools, if the template was updated using v2.8 I think this is expected

Reply:
nf-core/tools version 2.8 uses 22.10.1, see https://github.com/nf-core/tools/blob/0912990a63ef29e44e07cc2ba6ab81113684e0ae/nf_core/pipeline-template/nextflow.config#L225

Reply:
The Nextflow version will change in v2.9, it's required in order to use `singularity.registry` https://github.com/nf-core/tools/pull/2305

Reply:
Alright, so we are adding here nf-core/tools dev changes instead of 2.8? Why? That isnt at all my goal.

Reply:
Well, whatever works.